src/test/java/pageObjects/listingDetails_page.java

```diff
@@ -0,0 +1,74 @@
+package pageObjects;
+
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.By;
+
+import org.openqa.selenium.WebDriver;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import stepDefinitions.hooks;
+
+
+public class listingDetails_page {
+	
+	private WebDriver driver;
+	private static WebElement element = null;
+	
+	
+	public static void load_listings_details_page(WebDriver driver) {
+			driver = hooks.driver;
+			driver.get("https://chpl.ahrqstg.org/#/product/8481");
+			WebDriverWait wait = new WebDriverWait(driver, 20);
+			driver.manage().window().maximize();
+			String actualTitle = driver.getTitle();
+			System.out.println("page title is : "+actualTitle);
+		
+	}
+	
+	public static void sed_details_accordion_open(WebDriver driver){
+	
+		driver.get("https://chpl.ahrqdev.org/#/product/8511");	
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")));
+		
```

Comment:
The first lines here should be part of the method above where we get to the page. This method is there to "open the sed section of the accordion"; it should not be navigating anywhere.